azure-iot-device/tests/iothub/test_sync_clients.py

```diff
@@ -76,11 +76,17 @@ class SharedClientCreateFromConnectionStringTests(object):
     @pytest.mark.parametrize(
         "ca_cert",
         [
-            pytest.param(None, id="No CA certificate"),
-            pytest.param("some-certificate", id="With CA certificate"),
+            pytest.param(None, id=" No CA certificate "),
+            pytest.param("some-certificate", id=" With CA certificate "),
         ],
     )
-    def test_auth_provider_creation(self, mocker, client_class, connection_string, ca_cert):
+    @pytest.mark.parametrize(
+        "websockets",
+        [pytest.param(None, id=" No Websockets "), pytest.param(True, id=" With Websockets ")],
```

Comment:
There should also be a category of False. You want to test:

1) Default - No Websockets
2) Websockets == True
3) Websockets == False

Reply:
done.